blogs/2023/07/20/mangling-vscode.md

```diff
@@ -0,0 +1,174 @@
+---
+Order: 83
+TOCTitle: How we cut 20% off the size of VS Code's JavaScript
```

Comment:
This is pretty long for a TOC title and would probably spill over three lines.
Maybe "VS Code and JS name mangling" or "How we shrunk VS Code by 20%"